tools/llm_bench/task/visual_language_generation.py

```diff
@@ -211,7 +223,10 @@ def run_visual_language_generation_genai(
     gen_config.do_sample = False
     gen_config.ignore_eos = True
     kwargs = {}
-    if len(images) >= 1:
+    if len(images) > 1:
+        # multi images
+        kwargs["images"] = images
+    elif len(images) == 1:
         kwargs["images"] = images[0]
```

Comment:
I think It's ok to use just `kwargs["images"] = images` for one image too

```suggestion
    if len(images) >= 1:
        kwargs["images"] = images
```

Reply:
Updated, can you review again?